gulpfile.js

```diff
@@ -146,14 +146,14 @@ gulp.task( 'uglify', [ 'requirejs' ], function() {
 
 // ----- css ----- //
 
-var minifyCSS = require('gulp-minify-css');
+var cleanCSS = require('gulp-clean-css');
 
 gulp.task( 'css', function() {
   gulp.src('css/flickity.css')
     // copy to dist
     .pipe( gulp.dest('dist') )
     // minify
-    .pipe( minifyCSS({ advanced: false }) )
+    .pipe( cleanCSS({ advanced: false }) )
```

Comment:
No problem, in the past I've used `gulp-cssnano` but it seems like `gulp-clean-css` is far more used, so I assumed it was probably a better option. True, I probably should have made a pull request, rookie mistakes. Ha ha.